Corona-Warn-App/src/main/java/de/rki/coronawarnapp/covidcertificate/vaccination/ui/cards/VaccinationCard.kt

```diff
@@ -26,12 +27,17 @@ class VaccinationCard(parent: ViewGroup) :
         personName.text = curItem.vaccinatedPerson.fullName
         when (curItem.vaccinatedPerson.getVaccinationStatus()) {
             VaccinatedPerson.Status.COMPLETE -> {
-                val days = curItem.vaccinatedPerson.getTimeUntilImmunity()!!.standardDays.toInt()
-                vaccinationState.text = context.resources.getQuantityString(
-                    R.plurals.vaccination_card_status_vaccination_complete,
-                    days,
-                    days
-                )
+                val immunityIn = curItem.vaccinatedPerson.getTimeUntilImmunity()!!
+                vaccinationState.text = if (immunityIn < Duration.standardDays(1)) {
+                    resources.getString(R.string.vaccination_card_status_vaccination_complete_tomorrow)
+                } else {
+                    val days = immunityIn.standardDays.toInt()
+                    resources.getQuantityString(
+                        R.plurals.vaccination_card_status_vaccination_complete,
+                        days,
+                        days
+                    )
+                }
```

Comment:
Final shot at `2021-06-14`:

* `2021-06-27` -> `Vollständiger Impfschutz in 1 Tag`
* `2021-06-28` -> `Vollständiger Impfschutz ab morgen`
* `2021-06-29` -> IMMUNITY Card

Reply:
Just that I get it right, this fixes #3561, right?

Reply:
I think that for most people (especially non-programmers) the sentences “Vollständiger Impfschutz in 1 Tag” and “Vollständiger Impfschutz ab morgen” are equivalent. Therefore I think the message on 2021-06-27 is incorrect and should read “Vollständiger Impfschutz in 2 Tagen”.

Reply:
Yes, I think so.
Once `immunityIn < Duration.standardDays(1)` is `true`, `tomorrow` will be displayed.

Reply:
> I think that for most people (especially non-programmers) the sentences “Vollständiger Impfschutz in 1 Tag” and “Vollständiger Impfschutz ab morgen” are equivalent. Therefore I think the message on 2021-06-27 is incorrect and should read “Vollständiger Impfschutz in 2 Tagen”.

@mawis That would mean: Display "tomorrow" when `immunityIn < Duration.standardDays(2)`, I'm partial to both, would have to check with UX team in any case.

Reply:
No, in 2 days stays in 2 days but in 1 day has to be tomorrow, if I understood @mawis correctly.

Reply:
Yes. Probably that's the point, that non-programmers don't think the way integers work, but normally round up to the next full day. So at 23:59 “in 1 day” at least to me still means “the next day”, i.e. at that moment that's ”in one minute”.

So given that indeed the calculation of `immunityIn` could be changed. Anyway I think `immunityIn` could stay the same as it is (working the normal way with integers), but the formatting should be changed. Therefore I personally would append a `+ 1` at the end of line 34 in the diff above.